libraries/AP_Networking/lwip_hal/arch/sys_arch.cpp

```diff
@@ -4,6 +4,9 @@
  */
 
 #include <AP_HAL/AP_HAL.h>
+#include <AP_Networking/AP_Networking_Config.h>
```

Comment:
Our pattern should have this at the top.  (i.e.e the HAL.h inlude shouldn't be there)

... also, our usual naming scheme for those config files has a lower-case `c` .